pkg/client/core/api/rest/github.go

```diff
@@ -20,6 +20,15 @@ type githubAPI struct {
 	out          io.Writer
 }
 
+func (a *githubAPI) GetGithubTeamMembers(opts client.GetGithubTeamMembers) (*[]client.GithubTeamMember, error) {
```

Comment:
You probably want to return a slice of pointers to a GithubTeamMember, and not a pointer to a slice?